test/remote/gateways/remote_rapyd_test.rb

```diff
@@ -145,6 +145,13 @@ def test_successful_purchase_with_reccurence_type
     assert_equal 'SUCCESS', response.message
   end
 
+  def test_successful_purchase_with_save_payment_method
+    @options[:pm_type] = 'gb_visa_mo_card'
+    response = @gateway.purchase(@amount, @credit_card, @options.merge(save_payment_method: true))
+    assert_success response
+    assert_equal 'SUCCESS', response.message
```

Comment:
Is there any additional assertion we can add to validate that Rapyd is receiving the field? Not sure if or how their response differs when the field is present or absent in the request, but if there's something we could check related to that, it would make the test a little stronger

Reply:
Good idea, will look at their docs and our responses.